mex-wholebodymodel/matlab/utilities/jointLimits.m

```diff
@@ -0,0 +1,5 @@
+%% Joint limits definition for icubGazeboSim (rad)
```

Comment:
You should use the `wbm_jointLimits` function in place of this hardcoded limits.